File: src/main/java/com/bar/example/TestController.java
package com.bar.example;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.ApplicationArguments;
import org.springframework.boot.ApplicationRunner;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/")
public class TestController implements ApplicationRunner {

    private TestRepository testRepository;

    public TestController(@Autowired TestRepository testRepository) {
        this.testRepository = testRepository;
    }

    @GetMapping
    public String hello() {
        return "Hellooooo";
    }

    @GetMapping("/ent")
    public List<Ent> ent() {
        return testRepository.findAll();
    }

    @Override
    public void run(ApplicationArguments args) throws Exception {
        Ent ent = new Ent("hello", 100);
        testRepository.save(ent);
    }
}

File: src/main/kotlin/com/example/bar/TestController.kt
package com.example.bar

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import reactor.core.publisher.Flux
import reactor.core.publisher.Mono

@RestController
@RequestMapping("/")
class TestController(val entRepository: EntRepository) {

    @GetMapping
    fun hello(): Mono<String> {
        return Mono.just("Hellooooo")
    }

    @GetMapping("/ent")
    fun ent(): Flux<Ent> {
        return entRepository.findAll()
    }
}